config/SupportedVehicles.xml

```diff
@@ -96,6 +96,33 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.018 1.39 1.352" width="2.195" height="4.48" length="7.78"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_920/rpx920.xml">
+            <loadingArea offset="0.095 1.39 0.816" width="2.195" height="4.48" length="8.662"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_1226/rpx1226.xml">
+            <loadingArea offset="0.059 1.39 -0.657" width="2.195" height="4.48" length="11.831"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
+            <loadingArea offset="-0.02 0.978 -2.661" width="1.991" height="2.642" length="5.268"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSEL10006/LSEL10006_Square.xml">
+            <loadingArea offset="0.015 1.137 -0.161" width="2.352" height="5.301" length="9.627"/>
```

Comment:
Same height issues:

![image](https://user-images.githubusercontent.com/15822357/165583701-fef2c22c-fa7b-46ce-a699-1f49e76f1229.png)

Reply:
I think we want to move the offset to: ```offset="0 1.137 -0.161"``` It's pretty rare we need to move the offset in the x-direction. You can see here, it slightly has moved it under one side of the edge of the trailer on the left:

![image](https://user-images.githubusercontent.com/15822357/165584156-f79f2932-7fd0-47fb-b89f-4bf941c6f1ff.png)